README.md

```diff
@@ -26,20 +26,45 @@ $ rails db:reseed (handled by mongoid)
 $ rails db:version
 ```
 
+If you want to use output migration use the hook `after_migrate`
+```
+Mongoid::Migration.after_migrate = ->(output, name, direction, crash) {
+  upload_to_s3(name, output, direction) if crash == false
+}
+```
+
 To override the default migrations path (`db/migrate`), add the following line to your `application.rb` file:
 ```
 Mongoid::Migrator.migrations_path = ['foo/bar/db/migrate', 'path/to/db/migrate']
 ```
 
-If you want to use output migration use the hook `after_migrate`
+## Multi databases support
+
+Default behavior is to store migrations in the `default` client database but for projects that require horizontal scalabilty, this gem supports migrations [sharding](https://en.wikipedia.org/wiki/Shard_\(database_architecture\)).
+
+To generate a migration that can be run on shards, suffix the migration generator command with `--shards` like:
+
 ```
-Mongoid::Migration.after_migrate = ->(output, name, direction, crash) {
-  upload_to_s3(name, output, direction) if crash == false
-}
+$ rails generate mongoid:migration <your_migration_name_here> --shards
 ```
 
+The migration will be created and stored in `db/migrate/shards` folder.
+
+In order to manage a sharded migration, run tasks with the `MONGOID_CLIENT_NAME` environment variable:
+
+```
+$ rails db:migrate MONGOID_CLIENT_NAME=shard42
+```
+
+The shards migrations will be executed and entries will be added in the `shard42` database.
```

Comment:
Thanks for this new section, it's much better :)
To be precise, it's not the shard42 "database" but the mongodb (hostname+port+database) referenced by the "shard42" client in mongoid.yml. Maybe you could also add a small mongoid.yml example with default + 2 shards to make this clearer.

Reply:
Good point, I didn't want to make this section too big but you're right that will be clearer with an example

Reply:
@jarthod I've updated the readme here: https://github.com/adacosta/mongoid_rails_migrations/pull/57/commits/8cc5f8a4c355eeb8b68e6072ac4e42fac4178c7f

Reply:
Look good, thanks!